internal/config/config.go

```diff
@@ -232,6 +232,9 @@ const (
 	SSHPrivateKeyPath = "SSH_PRIVATE_KEY_PATH"
 
 	SystemdNotifySocket = "NOTIFY_SOCKET"
+
+	// For test
+	ForbiddenFallbackToFullbackup = "FORBIDDEN_FALLBACK_TO_FULLBACKUP"
```

Comment:
Probably, this feature may be useful in production... Let's add it to documentation (both Pg/Gp).